### ISSUE TITLE
Teamtalk crash when change tab in preferences

### ISSUE THREAD

**Issue by math65** [BearWare/TeamTalk5#2130]
### Description

When you change tabs in Teamtalk 5.15 preferences on Mac, the application crashes and closes.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [X] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

You can change tab in preferences

### Actual behavior

Teamtalk crash and close

### Steps to reproduce problem

Open preferences, and change tab.

**Comment by vlad-a-c** [comment]
it happens almost on every dialog. when you open the connection dialog it also crashes and so does it do the same when you open a new client instance. basically it's unusable. i'm using macos sonoma 14.2.1 on apple silicon

**Comment by bear101** [comment]
I cannot reproduce the issue on my Macs (both Intel though).

**Comment by CoBC** [comment]
I'm wondering if it's not an issue with VoiceOver in Qt6.6.1, maybe try 
to reproduce the issue with VO enable...

**Comment by bear101** [comment]
Hm, yes, it looks related to VoiceOver. Now it also crashes on mine.

**Comment by CoBC** [comment]
I guess we have to determine if the issue is present only since Qt6.6.1 
or also in 6.6.0, and than report the issue to Qt... If it's related to 
6.6.1, maybe we can rollback to 6.6 until a fix.

**Comment by bear101** [comment]
Hm, I tried with Qt 6.7.0 beta1 and here it does not crash. If I navigate slowly in the channels tree then it also speaks the items.

**Comment by bear101** [comment]
Strange that they haven't mentioned this in the feature list.

**Comment by bear101** [comment]
@math65 @vlad-a-c Please try this version and see if it fixes the problem: https://bearware.dk/beta/TeamTalk5_v5.15_Setup_Qt6.7.0beta1.pkg

**Comment by bear101** [comment]
Probably the same crash issue: https://bugreports.qt.io/browse/QTBUG-119512

**Comment by bear101** [comment]
@vlad-a-c isn't accessibility working better in the channels tree for you as well? If I navigate slowly then VoiceOver speaks the items' properties.

Does this mean we can close #198?

**Comment by bear101** [comment]
Fixed in v5.15.1

**Comment by CoBC** [comment]
The accessibility in trees was already better with 6.6 if I remember 
correctly

**Comment by bear101** [comment]
Ah, ok. Then we should have put it in the ChangeLog :)
